airflow/config_templates/default_test.cfg

```diff
@@ -100,6 +100,19 @@ flower_port = 5555
 default_queue = default
 sync_parallelism = 0
 
+[mesos]
+master = localhost:5050
+framework_name = Airflow
+task_cpu = 1
+task_memory = 20
+checkpoint = False
+authenticate = False
+docker_image_slave = test/docker-airflow
+docker_volume_dag_driver = "local"
+docker_volume_dag_name = "airflowdags"
+docker_volume_dag_container_path = "/usr/local/airflow/dags"
+docker_sock = "/var/run/docker.sock"
```

Comment:
Anything that is the default value please don't specify in the test config.

Reply:
ah ok. :-) Sorry!